### PR TITLE
Update cryptex attribute IANA registration

### LINKED ISSUE #35: [AD Evaluation comments]  Missing IANA registration fields for  Section 4
(3) Section 4 defines a new SDP attribute, referencing RFC 4566 (replaced
by RFC8866), which requires that attributes be registered with IANA, but
this document doesn't appear to do that.  RFC8866 specifies 12 fields
required for such a registration, but this document provides only five.

### PR DESCRIPTION
**Pull request by murillo128**
closes #35

### CODE REVIEW

File: draft-ietf-avtcore-cryptex.md [hunk 1]
```diff
@@ -344,6 +328,32 @@ IANA Considerations
 
 This document defines two new 'defined by profile' attributes, as noted in RTP Header Processing.
 
+## New cryptex Media-Level SDP Attribute
+
+This document updates the "Session Description Protocol Parameters" registry as specified in Section 8.2.4 of {{RFC8866}}. Specifically, it adds the SDP 'crytpes' attribute to the table for SDP media-level attributes.
```
Comment (juberti) [inline draft-ietf-avtcore-cryptex.md:333]:
```suggestion
This document updates the "Session Description Protocol Parameters" registry as specified in Section 8.2.4 of {{RFC8866}}. Specifically, it adds the SDP 'cryptex' attribute to the table for SDP media-level attributes.
```

File: draft-ietf-avtcore-cryptex.md [hunk 2]
```diff
@@ -344,6 +328,32 @@ IANA Considerations
...
+    
+Attribute name: cryptex
+    
+Attribute syntax: This attribute takes no values.
+    
+Attribute semantics: N/A.
+    
+Attribute value: N/A.
+    
+Usage level: media.
```
Comment (juberti) [inline draft-ietf-avtcore-cryptex.md:347]:
```suggestion
Usage level: media-level
```

File: draft-ietf-avtcore-cryptex.md [hunk 3]
```diff
@@ -344,6 +328,32 @@ IANA Considerations
...
+
+Contact name: <TBD>
+    
+Contact email address: <TBD>
+    
+Attribute name: cryptex
+    
+Attribute syntax: This attribute takes no values.
+    
+Attribute semantics: N/A.
```
Comment (juberti) [inline draft-ietf-avtcore-cryptex.md:343]: I would recommend only using periods when it's a complete sentence.

Comment (murillo128) [reply draft-ietf-avtcore-cryptex.md]: I have removed then

**Comment by murillo128** [comment]
@juberti are you ok the PR now? Will like to get this merged and have a new draft published before cut off date today

**Comment by juberti** [comment]
lgtm

**Pull request merged**